static/app/views/setupWizard/wizardProjectSelection.tsx

```diff
@@ -277,141 +278,139 @@ export function WizardProjectSelection({
+              triggerProps={{
+                icon: isCreateProjectSelected ? (
+                  <IconAdd isCircled />
+                ) : selectedProject ? (
+                  <ProjectBadge avatarSize={16} project={selectedProject} hideName />
+                ) : null,
+                children: isCreateProjectSelected
+                  ? t('Create Project')
+                  : selectedProject?.slug || (
+                      <SelectPlaceholder>{t('Select a project')}</SelectPlaceholder>
+                    ),
+              }}
+              onChange={({value}) => {
+                setSelectedProjectId(value as string);
+              }}
+              emptyMessage={emptyMessage}
+              menuFooter={
+                isCreationEnabled
+                  ? ({closeOverlay}) => (
+                      <Flex justify="end">
+                        <Button
+                          size="xs"
+                          onClick={() => {
+                            setSelectedProjectId(CREATE_PROJECT_VALUE);
+                            closeOverlay();
+                          }}
+                          icon={<IconAdd isCircled />}
+                        >
+                          {t('Create Project')}
+                        </Button>
+                      </Flex>
+                    )
+                  : undefined
+              }
+            />
+          )}
+        </FieldWrapper>
+        {isCreateProjectSelected &&
+          (isOrgMemberWithNoAccess ? (
+            projectNameField
+          ) : (
+            <Columns>
+              {projectNameField}
+              <FieldWrapper>
+                <label>{t('Team')}</label>
+                <StyledCompactSelect
+                  value={newProjectTeam as string}
+                  options={
+                    selectableTeams?.map(team => ({
+                      value: team.slug,
+                      label: `#${team.slug}`,
+                      leadingItems: <IdBadge team={team} hideName />,
+                      searchKey: team.slug,
+                    })) || []
+                  }
+                  triggerProps={{
+                    icon: selectedTeam ? (
+                      <IdBadge avatarSize={16} team={selectedTeam} hideName />
+                    ) : null,
+                    children: selectedTeam ? `#${selectedTeam.slug}` : t('Select a team'),
+                  }}
+                  onChange={({value}) => {
+                    setNewProjectTeam(value as string);
+                  }}
+                />
+              </FieldWrapper>
+            </Columns>
+          ))}
+        <SubmitButton
+          disabled={!isFormValid || isPending}
+          priority="primary"
+          type="submit"
+        >
+          {t('Continue')}
+        </SubmitButton>
+      </Stack>
+    </form>
   );
 }
 
-const StyledForm = styled('form')`
-  display: flex;
-  flex-direction: column;
-  gap: ${space(2)};
-`;
-
```

Comment:
It did okay with this, but maybe we should expand `as` support?

Reply:
I'm not confident enough to say that we do, so let me punt this decision down the line a bit. We may want to have some built in functionality into a `Form` component that we don't want to or cant  build into a container